packages/dev-env/src/test-env.ts

```diff
@@ -191,11 +191,17 @@ export const runBsky = async (cfg: BskyConfig): Promise<BskyServerInfo> => {
     schema: cfg.dbPostgresSchema,
   })
 
+  // Separate migration db in case migration changes some connection state that we need in the tests, e.g. "alter table ... set ..."
```

Comment:
```suggestion
  // Separate migration db in case migration changes some connection state that we need in the tests, e.g. "alter database ... set ..."
```